<?xml version='1.1' encoding='UTF-8'?>
<result plugin="junit@1.53">
  <suites>
    <suite>
      <file>/var/jenkins_home/workspace/fdc-api@2/target/surefire-reports/TEST-TestSuite.xml</file>
      <name>TestSuite</name>
      <duration>199.918</duration>
      <time>199.918</time>
      <nodeId>19</nodeId>
      <enclosingBlocks>
        <string>15</string>
      </enclosingBlocks>
      <enclosingBlockNames>
        <string>Buid Jar aftifacts</string>
      </enclosingBlockNames>
      <cases>
        <case>
          <duration>1.896</duration>
          <className>com.fdc.core.repository.EventsRepositoryIntTest</className>
          <testName>findByEventPackageAndEventTypeTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.1</duration>
          <className>com.fdc.core.repository.EventsRepositoryIntTest</className>
          <testName>findByIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.132</duration>
          <className>com.fdc.core.repository.EventsRepositoryIntTest</className>
          <testName>getPackageStatusDetailsTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.033</duration>
          <className>com.fdc.core.repository.OptimizationRepositoryIntTest</className>
          <testName>findByIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.025</duration>
          <className>com.fdc.core.repository.OptimizationRepositoryIntTest</className>
          <testName>findByOptimizationIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.031</duration>
          <className>com.fdc.core.repository.OptimizationRepositoryIntTest</className>
          <testName>findByParcelCenterIdAndDeliveryDateAndActiveTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.013</duration>
          <className>com.fdc.core.repository.ClientRepositoryIntTest</className>
          <testName>findByIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.012</duration>
          <className>com.fdc.core.repository.RouteRepositoryIntTest</className>
          <testName>findByIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.037</duration>
          <className>com.fdc.core.repository.RouteRepositoryIntTest</className>
          <testName>findByR4meRouteIdAndOptimizationIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.021</duration>
          <className>com.fdc.core.repository.RouteRepositoryIntTest</className>
          <testName>findByR4meRouteIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.311</duration>
          <className>com.fdc.core.repository.RouteRepositoryIntTest</className>
          <testName>updateRouteStatusAsStartedTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.042</duration>
          <className>com.fdc.core.repository.PackagesRepositoryIntTest</className>
          <testName>findByBarcodeAndTrackingNumberTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.063</duration>
          <className>com.fdc.core.repository.PackagesRepositoryIntTest</className>
          <testName>findByBarcodeTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.048</duration>
          <className>com.fdc.core.repository.PackagesRepositoryIntTest</className>
          <testName>findByIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.041</duration>
          <className>com.fdc.core.repository.PackagesRepositoryIntTest</className>
          <testName>findByR4meOrderIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.024</duration>
          <className>com.fdc.core.repository.PackagesRepositoryIntTest</className>
          <testName>findByR4meSyncStatusAndPcIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.023</duration>
          <className>com.fdc.core.repository.PackagesRepositoryIntTest</className>
          <testName>findByRouteTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.037</duration>
          <className>com.fdc.core.repository.PackagesRepositoryIntTest</className>
          <testName>findPackagesInRouteTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.041</duration>
          <className>com.fdc.core.repository.PackagesRepositoryIntTest</className>
          <testName>findWithShipmentAndClientByR4meSyncStatusAndPcIdAndDeliveryDateTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.08</duration>
          <className>com.fdc.core.repository.PackagesRepositoryIntTest</className>
          <testName>findWithShipmentAndRouteByR4meSyncStatusAndPcIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.031</duration>
          <className>com.fdc.core.repository.PackagesRepositoryIntTest</className>
          <testName>findWithShipmentByIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.046</duration>
          <className>com.fdc.core.repository.PackagesRepositoryIntTest</className>
          <testName>findWithShipmentByR4meSyncStatusAndPcIdAndDeliveryDateTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.036</duration>
          <className>com.fdc.core.repository.PackagesRepositoryIntTest</className>
          <testName>findWithShipmentByR4meSyncStatusAndPcIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.099</duration>
          <className>com.fdc.core.repository.PackagesRepositoryIntTest</className>
          <testName>findWithShipmentByR4meSyncStatusInAndPcIdAndDeliveryDateTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.07</duration>
          <className>com.fdc.core.repository.PackagesRepositoryIntTest</className>
          <testName>findWithShipmentByR4meSyncStatusInAndPcIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.066</duration>
          <className>com.fdc.core.repository.PackagesRepositoryIntTest</className>
          <testName>updatePackageStatusInBulkTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.035</duration>
          <className>com.fdc.core.repository.PackagesRepositoryIntTest</className>
          <testName>updatePackageStatusTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.008</duration>
          <className>com.fdc.core.repository.PcCustomAttributeRepositoryIntTest</className>
          <testName>findByIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.038</duration>
          <className>com.fdc.core.repository.PcCustomAttributeRepositoryIntTest</className>
          <testName>findByParcelCenterIdAndFieldNameTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.09</duration>
          <className>com.fdc.core.repository.PcCustomAttributeRepositoryIntTest</className>
          <testName>findByParcelCenterIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.004</duration>
          <className>com.fdc.core.repository.ZipcodeCoverageRepositoryIntTest</className>
          <testName>findByIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.006</duration>
          <className>com.fdc.core.repository.ZipcodeCoverageRepositoryIntTest</className>
          <testName>findWithParcelCenterByZipcodeStartingWithTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.003</duration>
          <className>com.fdc.core.repository.EventTypeRepositoryIntTest</className>
          <testName>findByIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.004</duration>
          <className>com.fdc.core.repository.DriverRepositoryIntTest</className>
          <testName>findByIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.008</duration>
          <className>com.fdc.core.repository.DriverRepositoryIntTest</className>
          <testName>findByMemberIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.013</duration>
          <className>com.fdc.core.repository.ParcelCenterRepositoryIntTest</className>
          <testName>findAllWithPcMembersTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.012</duration>
          <className>com.fdc.core.repository.ParcelCenterRepositoryIntTest</className>
          <testName>findByIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.015</duration>
          <className>com.fdc.core.repository.ParcelCenterRepositoryIntTest</className>
          <testName>findWithPcMembersByIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.019</duration>
          <className>com.fdc.core.repository.ShipmentRepositoryIntTest</className>
          <testName>findByExternalShipmentIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.013</duration>
          <className>com.fdc.core.repository.ShipmentRepositoryIntTest</className>
          <testName>findByIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.012</duration>
          <className>com.fdc.core.repository.ParcelCenterMemberRepositoryIntTest</className>
          <testName>findByIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.049</duration>
          <className>com.fdc.core.repository.ParcelCenterMemberRepositoryIntTest</className>
          <testName>findByParcelCenterIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.004</duration>
          <className>com.fdc.core.shipment.processing.post.async.dell.DellShipmentAsyncPostProcessingRunnerIntTest</className>
          <testName>processors_present</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>2.454</duration>
          <className>com.fdc.core.controller.ShipmentControllerIntTest</className>
          <testName>fetchShipmentByExtIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.029</duration>
          <className>com.fdc.core.controller.ShipmentControllerIntTest</className>
          <testName>fetchShipmentByIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>1.858</duration>
          <className>com.fdc.core.controller.ShipmentControllerIntTest</className>
          <testName>insertShipmentInBulkTest_emptyAddressLine1</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.086</duration>
          <className>com.fdc.core.controller.ShipmentControllerIntTest</className>
          <testName>insertShipmentInBulkTest_emptyCity</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.148</duration>
          <className>com.fdc.core.controller.ShipmentControllerIntTest</className>
          <testName>insertShipmentInBulkTest_emptyList</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.149</duration>
          <className>com.fdc.core.controller.ShipmentControllerIntTest</className>
          <testName>insertShipmentInBulkTest_emptyZip</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.194</duration>
          <className>com.fdc.core.controller.ShipmentControllerIntTest</className>
          <testName>insertShipmentInBulkTest_esmptyState</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.271</duration>
          <className>com.fdc.core.controller.ShipmentControllerIntTest</className>
          <testName>insertShipmentTest_emptyAddressLine1</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.147</duration>
          <className>com.fdc.core.controller.ShipmentControllerIntTest</className>
          <testName>insertShipmentTest_emptyCity</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.601</duration>
          <className>com.fdc.core.controller.ShipmentControllerIntTest</className>
          <testName>insertShipmentTest_emptyState</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.334</duration>
          <className>com.fdc.core.controller.ShipmentControllerIntTest</className>
          <testName>insertShipmentTest_emptyZip</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.198</duration>
          <className>com.fdc.core.controller.ShipmentControllerIntTest</className>
          <testName>insertShipmentTest_nullDeliveryDate</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.074</duration>
          <className>com.fdc.core.controller.ShipmentControllerIntTest</className>
          <testName>patchShipmentTest_emptyAddressLine1</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.149</duration>
          <className>com.fdc.core.controller.ShipmentControllerIntTest</className>
          <testName>patchShipmentTest_emptyCity</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.204</duration>
          <className>com.fdc.core.controller.ShipmentControllerIntTest</className>
          <testName>patchShipmentTest_emptyState</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.026</duration>
          <className>com.fdc.core.controller.ShipmentControllerIntTest</className>
          <testName>patchShipmentTest_emptyZip</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.247</duration>
          <className>com.fdc.core.controller.ShipmentControllerIntTest</className>
          <testName>patchShipmentTest_nullDeliveryDate</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.09</duration>
          <className>com.fdc.core.controller.ShipmentControllerIntTest</className>
          <testName>updateShipmentTest_emptyAddressLine1</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.087</duration>
          <className>com.fdc.core.controller.ShipmentControllerIntTest</className>
          <testName>updateShipmentTest_emptyCity</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.342</duration>
          <className>com.fdc.core.controller.ShipmentControllerIntTest</className>
          <testName>updateShipmentTest_emptyState</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.057</duration>
          <className>com.fdc.core.controller.ShipmentControllerIntTest</className>
          <testName>updateShipmentTest_emptyZip</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.062</duration>
          <className>com.fdc.core.controller.ShipmentControllerIntTest</className>
          <testName>updateShipmentTest_nullDeliveryDate</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.002</duration>
          <className>com.fdc.core.event.processor.impl.OrderCreatedEventProcessorIntTest</className>
          <testName>testProcessEvent</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.317</duration>
          <className>com.fdc.core.event.processor.impl.RouteOptimizedEventProcessorIntTest</className>
          <testName>processEventTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.03</duration>
          <className>com.fdc.core.event.processor.impl.RouteOwnerChangedEventProcessorIntTest</className>
          <testName>processEventTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>1.131</duration>
          <className>com.fdc.core.event.processor.impl.DeliveredEventProcessorIntTest</className>
          <testName>testWorkFlowId2SequenceEvents</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.007</duration>
          <className>com.fdc.core.service.r4me.impl.R4meAddressServiceImplUnitTest</className>
          <testName>getR4meAddressesFromParcelCenterAndPackagesTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.527</duration>
          <className>com.fdc.core.service.impl.ShipmentServiceImplClientPickupTest</className>
          <testName>test_insertShipment_withClientPickup</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.017</duration>
          <className>com.fdc.core.service.impl.OptimizationServiceImplUnitTest</className>
          <testName>createOptimizationTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.01</duration>
          <className>com.fdc.core.service.impl.OptimizationServiceImplUnitTest</className>
          <testName>deactivateOptimizationTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.01</duration>
          <className>com.fdc.core.service.impl.ZipcodeCoverageServiceImplUnitTest</className>
          <testName>getParcelCenterByZipcodeTest_notFound</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.008</duration>
          <className>com.fdc.core.service.impl.EventServiceImplUnitTest</className>
          <testName>captureDepartedFromOriginEventTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.018</duration>
          <className>com.fdc.core.service.impl.EventServiceImplUnitTest</className>
          <testName>captureInfoReceivedEventTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.013</duration>
          <className>com.fdc.core.service.impl.PackageServiceImplUnitTest</className>
          <testName>validatePackagesForPatchTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.003</duration>
          <className>com.fdc.core.service.impl.PackageServiceImplUnitTest</className>
          <testName>validatePackagesTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.002</duration>
          <className>com.fdc.core.service.impl.ShipmentServiceImplUnitTest</className>
          <testName>fetchShipmentByExtIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.001</duration>
          <className>com.fdc.core.service.impl.ShipmentServiceImplUnitTest</className>
          <testName>fetchShipmentByExtIdTest_notFound</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.004</duration>
          <className>com.fdc.core.service.impl.ShipmentServiceImplUnitTest</className>
          <testName>fetchShipmentByIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.003</duration>
          <className>com.fdc.core.service.impl.ShipmentServiceImplUnitTest</className>
          <testName>fetchShipmentByIdTest_notFound</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.016</duration>
          <className>com.fdc.core.service.impl.ShipmentServiceImplUnitTest</className>
          <testName>insertShipmentTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.009</duration>
          <className>com.fdc.core.service.impl.ShipmentServiceImplUnitTest</className>
          <testName>insertShipmentsInBulkTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.004</duration>
          <className>com.fdc.core.service.impl.ShipmentServiceImplUnitTest</className>
          <testName>patchShipmentTest_allFields</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.005</duration>
          <className>com.fdc.core.service.impl.ShipmentServiceImplUnitTest</className>
          <testName>patchShipmentTest_singleField</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.005</duration>
          <className>com.fdc.core.service.impl.ShipmentServiceImplUnitTest</className>
          <testName>updateShipmentTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.004</duration>
          <className>com.fdc.core.service.impl.ParcelCenterServiceImplUnitTest</className>
          <testName>findByIdTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.004</duration>
          <className>com.fdc.core.service.impl.ParcelCenterServiceImplUnitTest</className>
          <testName>findByIdTest_notFound</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.007</duration>
          <className>com.fdc.core.mapper.impl.ActivityToEventMapperImplUnitTest</className>
          <testName>mapFromActivityToEventTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.0</duration>
          <className>com.fdc.core.mapper.impl.ActivityToEventMapperImplUnitTest</className>
          <testName>mapFromActivityToEventTest_nullInput</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.001</duration>
          <className>com.fdc.core.mapper.impl.ActivityTimelineMapperImplUnitTest</className>
          <testName>mapEventToActivityDtoTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.001</duration>
          <className>com.fdc.core.mapper.impl.ActivityTimelineMapperImplUnitTest</className>
          <testName>mapEventToActivityDtoTest_nullInput</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.001</duration>
          <className>com.fdc.core.mapper.impl.ActivityTimelineMapperImplUnitTest</className>
          <testName>mapEventsToActivitiesTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.001</duration>
          <className>com.fdc.core.mapper.impl.ActivityTimelineMapperImplUnitTest</className>
          <testName>mapEventsToActivitiesTest_nullInput</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.009</duration>
          <className>com.fdc.core.mapper.mapstruct.ShipmentMapperUnitTest</className>
          <testName>mapFromDtoTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.016</duration>
          <className>com.fdc.core.mapper.mapstruct.ShipmentMapperUnitTest</className>
          <testName>mapToDtoTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.002</duration>
          <className>com.fdc.core.mapper.mapstruct.PackageMapperUnitTest</className>
          <testName>mapFromDtoTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.021</duration>
          <className>com.fdc.core.mapper.mapstruct.PackageMapperUnitTest</className>
          <testName>mapToDtoTest</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.001</duration>
          <className>com.fdc.core.spring.publishers.ShipmentCreatedEventPublisherUnitTest</className>
          <testName>publishOnShipmentCreation_nullShipmentResponseDto_exception</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.0</duration>
          <className>com.fdc.core.spring.publishers.ShipmentCreatedEventPublisherUnitTest</className>
          <testName>publishOnShipmentCreation_null_shipment_exception</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.017</duration>
          <className>com.fdc.core.spring.publishers.ShipmentCreatedEventPublisherUnitTest</className>
          <testName>publishOnShipmentCreation_ok</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.001</duration>
          <className>com.fdc.core.ShipmentsApplicationTest</className>
          <testName>contextLoads</testName>
          <skipped>false</skipped>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>4.774</duration>
          <className>com.fdc.core.cloudsort.service.auth.CloudSortJwtStorageIntTest</className>
          <testName>springTestContextPrepareTestInstance</testName>
          <skipped>false</skipped>
          <errorStackTrace>java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: com.github.tomakehurst.wiremock.common.FatalStartupException: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
Caused by: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
Caused by: java.io.IOException: Failed to bind to /0.0.0.0:8191
Caused by: java.net.BindException: Address already in use
</errorStackTrace>
          <errorDetails>Failed to load ApplicationContext</errorDetails>
          <stdout>
  .   ____          _            __ _ _
 /\\ / ___&apos;_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | &apos;_ | &apos;_| | &apos;_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  &apos;  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::                (v2.5.6)

2022-08-30 12:40:36.046 Verbose logging enabled
2022-08-30 12:40:37.542 WARN [main] wiremock.org.eclipse.jetty.server.AbstractConnector - Acceptors should be &lt;= availableProcessors: NetworkTrafficServerConnector@200ca1fe{HTTP/1.1, (http/1.1, h2c)}{0.0.0.0:0}
2022-08-30 12:40:40.013 ERROR [main] org.springframework.boot.SpringApplication - Application run failed
com.github.tomakehurst.wiremock.common.FatalStartupException: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:157) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.support.DefaultTestContext.getApplicationContext(DefaultTestContext.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:190) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.prepareTestInstance(ServletTestExecutionListener.java:132) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:248) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.springTestContextPrepareTestInstance(AbstractTestNGSpringContextTests.java:149) ~[spring-test-5.3.12.jar:5.3.12]
	at jdk.internal.reflect.GeneratedMethodAccessor300.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122) ~[testng-7.4.0.jar:7.4.0]
	at java.util.ArrayList.forEach(ArrayList.java:1541) [?:?]
	at org.testng.TestRunner.privateRun(TestRunner.java:794) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestRunner.run(TestRunner.java:596) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.run(SuiteRunner.java:276) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuites(TestNG.java:1063) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.run(TestNG.java:1031) [testng-7.4.0.jar:7.4.0]
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418) [surefire-booter-2.22.2.jar:2.22.2]
Caused by: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:220) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	... 42 more
Caused by: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:349) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	... 42 more
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:455) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:447) ~[?:?]
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227) ~[?:?]
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:80) ~[?:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:344) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	... 42 more
2022-08-30 12:40:40.027 ERROR [main] org.springframework.test.context.TestContextManager - Caught exception while allowing TestExecutionListener [org.springframework.test.context.web.ServletTestExecutionListener@8c6da7c] to prepare test instance [com.fdc.core.cloudsort.service.auth.CloudSortJwtStorageIntTest@7cf162bc]
java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:132) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.support.DefaultTestContext.getApplicationContext(DefaultTestContext.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:190) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.prepareTestInstance(ServletTestExecutionListener.java:132) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:248) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.springTestContextPrepareTestInstance(AbstractTestNGSpringContextTests.java:149) ~[spring-test-5.3.12.jar:5.3.12]
	at jdk.internal.reflect.GeneratedMethodAccessor300.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122) ~[testng-7.4.0.jar:7.4.0]
	at java.util.ArrayList.forEach(ArrayList.java:1541) [?:?]
	at org.testng.TestRunner.privateRun(TestRunner.java:794) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestRunner.run(TestRunner.java:596) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.run(SuiteRunner.java:276) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuites(TestNG.java:1063) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.run(TestNG.java:1031) [testng-7.4.0.jar:7.4.0]
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418) [surefire-booter-2.22.2.jar:2.22.2]
Caused by: com.github.tomakehurst.wiremock.common.FatalStartupException: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:157) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 35 more
Caused by: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:220) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 35 more
Caused by: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:349) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 35 more
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:455) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:447) ~[?:?]
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227) ~[?:?]
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:80) ~[?:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:344) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 35 more
</stdout>
          <failedSince>318</failedSince>
        </case>
        <case>
          <duration>5.168</duration>
          <className>com.fdc.core.cloudsort.service.auth.CloudSortJwtStorageIntTest</className>
          <testName>springTestContextBeforeTestMethod</testName>
          <skipped>false</skipped>
          <errorStackTrace>java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: com.github.tomakehurst.wiremock.common.FatalStartupException: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
Caused by: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
Caused by: java.io.IOException: Failed to bind to /0.0.0.0:8191
Caused by: java.net.BindException: Address already in use
</errorStackTrace>
          <errorDetails>Failed to load ApplicationContext</errorDetails>
          <stdout>
  .   ____          _            __ _ _
 /\\ / ___&apos;_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | &apos;_ | &apos;_| | &apos;_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  &apos;  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::                (v2.5.6)

2022-08-30 12:40:40.452 Verbose logging enabled
2022-08-30 12:40:40.463 WARN [main] wiremock.org.eclipse.jetty.server.AbstractConnector - Acceptors should be &lt;= availableProcessors: NetworkTrafficServerConnector@cdae79d{HTTP/1.1, (http/1.1, h2c)}{0.0.0.0:0}
2022-08-30 12:40:40.529 ERROR [main] org.springframework.boot.SpringApplication - Application run failed
com.github.tomakehurst.wiremock.common.FatalStartupException: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:157) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.support.DefaultTestContext.getApplicationContext(DefaultTestContext.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:190) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.beforeTestMethod(ServletTestExecutionListener.java:145) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.TestContextManager.beforeTestMethod(TestContextManager.java:293) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.springTestContextBeforeTestMethod(AbstractTestNGSpringContextTests.java:161) ~[spring-test-5.3.12.jar:5.3.12]
	at jdk.internal.reflect.GeneratedMethodAccessor160.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:527) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128) ~[testng-7.4.0.jar:7.4.0]
	at java.util.ArrayList.forEach(ArrayList.java:1541) [?:?]
	at org.testng.TestRunner.privateRun(TestRunner.java:794) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestRunner.run(TestRunner.java:596) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.run(SuiteRunner.java:276) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuites(TestNG.java:1063) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.run(TestNG.java:1031) [testng-7.4.0.jar:7.4.0]
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418) [surefire-booter-2.22.2.jar:2.22.2]
Caused by: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:220) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	... 48 more
Caused by: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:349) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	... 48 more
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:455) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:447) ~[?:?]
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227) ~[?:?]
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:80) ~[?:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:344) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	... 48 more
2022-08-30 12:40:40.538 WARN [main] org.springframework.test.context.TestContextManager - Caught exception while invoking &apos;beforeTestMethod&apos; callback on TestExecutionListener [org.springframework.test.context.web.ServletTestExecutionListener@8c6da7c] for test method [public void com.fdc.core.cloudsort.service.auth.CloudSortJwtStorageIntTest.get_expiredToken_ok()] and test instance [com.fdc.core.cloudsort.service.auth.CloudSortJwtStorageIntTest@7cf162bc]
java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:132) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.support.DefaultTestContext.getApplicationContext(DefaultTestContext.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:190) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.beforeTestMethod(ServletTestExecutionListener.java:145) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.TestContextManager.beforeTestMethod(TestContextManager.java:293) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.springTestContextBeforeTestMethod(AbstractTestNGSpringContextTests.java:161) ~[spring-test-5.3.12.jar:5.3.12]
	at jdk.internal.reflect.GeneratedMethodAccessor160.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:527) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128) ~[testng-7.4.0.jar:7.4.0]
	at java.util.ArrayList.forEach(ArrayList.java:1541) [?:?]
	at org.testng.TestRunner.privateRun(TestRunner.java:794) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestRunner.run(TestRunner.java:596) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.run(SuiteRunner.java:276) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuites(TestNG.java:1063) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.run(TestNG.java:1031) [testng-7.4.0.jar:7.4.0]
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418) [surefire-booter-2.22.2.jar:2.22.2]
Caused by: com.github.tomakehurst.wiremock.common.FatalStartupException: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:157) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 41 more
Caused by: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:220) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 41 more
Caused by: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:349) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 41 more
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:455) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:447) ~[?:?]
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227) ~[?:?]
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:80) ~[?:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:344) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 41 more
</stdout>
          <failedSince>318</failedSince>
        </case>
        <case>
          <duration>0.317</duration>
          <className>com.fdc.core.cloudsort.service.auth.CloudSortJwtStorageIntTest</className>
          <testName>springTestContextBeforeTestMethod</testName>
          <skipped>false</skipped>
          <errorStackTrace>java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: com.github.tomakehurst.wiremock.common.FatalStartupException: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
Caused by: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
Caused by: java.io.IOException: Failed to bind to /0.0.0.0:8191
Caused by: java.net.BindException: Address already in use
</errorStackTrace>
          <errorDetails>Failed to load ApplicationContext</errorDetails>
          <stdout>
  .   ____          _            __ _ _
 /\\ / ___&apos;_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | &apos;_ | &apos;_| | &apos;_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  &apos;  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::                (v2.5.6)

2022-08-30 12:40:40.758 Verbose logging enabled
2022-08-30 12:40:40.769 WARN [main] wiremock.org.eclipse.jetty.server.AbstractConnector - Acceptors should be &lt;= availableProcessors: NetworkTrafficServerConnector@13e5d994{HTTP/1.1, (http/1.1, h2c)}{0.0.0.0:0}
2022-08-30 12:40:40.825 ERROR [main] org.springframework.boot.SpringApplication - Application run failed
com.github.tomakehurst.wiremock.common.FatalStartupException: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:157) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.support.DefaultTestContext.getApplicationContext(DefaultTestContext.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:190) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.beforeTestMethod(ServletTestExecutionListener.java:145) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.TestContextManager.beforeTestMethod(TestContextManager.java:293) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.springTestContextBeforeTestMethod(AbstractTestNGSpringContextTests.java:161) ~[spring-test-5.3.12.jar:5.3.12]
	at jdk.internal.reflect.GeneratedMethodAccessor160.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:527) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128) ~[testng-7.4.0.jar:7.4.0]
	at java.util.ArrayList.forEach(ArrayList.java:1541) [?:?]
	at org.testng.TestRunner.privateRun(TestRunner.java:794) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestRunner.run(TestRunner.java:596) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.run(SuiteRunner.java:276) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuites(TestNG.java:1063) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.run(TestNG.java:1031) [testng-7.4.0.jar:7.4.0]
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418) [surefire-booter-2.22.2.jar:2.22.2]
Caused by: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:220) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	... 48 more
Caused by: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:349) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	... 48 more
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:455) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:447) ~[?:?]
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227) ~[?:?]
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:80) ~[?:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:344) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	... 48 more
2022-08-30 12:40:40.826 WARN [main] org.springframework.test.context.TestContextManager - Caught exception while invoking &apos;beforeTestMethod&apos; callback on TestExecutionListener [org.springframework.test.context.web.ServletTestExecutionListener@8c6da7c] for test method [public void com.fdc.core.cloudsort.service.auth.CloudSortJwtStorageIntTest.get_ok()] and test instance [com.fdc.core.cloudsort.service.auth.CloudSortJwtStorageIntTest@7cf162bc]
java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:132) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.support.DefaultTestContext.getApplicationContext(DefaultTestContext.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:190) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.beforeTestMethod(ServletTestExecutionListener.java:145) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.TestContextManager.beforeTestMethod(TestContextManager.java:293) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.springTestContextBeforeTestMethod(AbstractTestNGSpringContextTests.java:161) ~[spring-test-5.3.12.jar:5.3.12]
	at jdk.internal.reflect.GeneratedMethodAccessor160.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:527) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128) ~[testng-7.4.0.jar:7.4.0]
	at java.util.ArrayList.forEach(ArrayList.java:1541) [?:?]
	at org.testng.TestRunner.privateRun(TestRunner.java:794) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestRunner.run(TestRunner.java:596) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.run(SuiteRunner.java:276) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuites(TestNG.java:1063) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.run(TestNG.java:1031) [testng-7.4.0.jar:7.4.0]
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418) [surefire-booter-2.22.2.jar:2.22.2]
Caused by: com.github.tomakehurst.wiremock.common.FatalStartupException: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:157) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 41 more
Caused by: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:220) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 41 more
Caused by: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:349) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 41 more
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:455) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:447) ~[?:?]
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227) ~[?:?]
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:80) ~[?:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:344) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 41 more
</stdout>
          <failedSince>318</failedSince>
        </case>
        <case>
          <duration>0.241</duration>
          <className>com.fdc.core.cloudsort.service.auth.CloudSortJwtStorageIntTest</className>
          <testName>springTestContextBeforeTestMethod</testName>
          <skipped>false</skipped>
          <errorStackTrace>java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: com.github.tomakehurst.wiremock.common.FatalStartupException: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
Caused by: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
Caused by: java.io.IOException: Failed to bind to /0.0.0.0:8191
Caused by: java.net.BindException: Address already in use
</errorStackTrace>
          <errorDetails>Failed to load ApplicationContext</errorDetails>
          <stdout>
  .   ____          _            __ _ _
 /\\ / ___&apos;_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | &apos;_ | &apos;_| | &apos;_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  &apos;  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::                (v2.5.6)

2022-08-30 12:40:41.010 Verbose logging enabled
2022-08-30 12:40:41.025 WARN [main] wiremock.org.eclipse.jetty.server.AbstractConnector - Acceptors should be &lt;= availableProcessors: NetworkTrafficServerConnector@c409b0e{HTTP/1.1, (http/1.1, h2c)}{0.0.0.0:0}
2022-08-30 12:40:41.073 ERROR [main] org.springframework.boot.SpringApplication - Application run failed
com.github.tomakehurst.wiremock.common.FatalStartupException: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:157) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.support.DefaultTestContext.getApplicationContext(DefaultTestContext.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:190) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.beforeTestMethod(ServletTestExecutionListener.java:145) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.TestContextManager.beforeTestMethod(TestContextManager.java:293) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.springTestContextBeforeTestMethod(AbstractTestNGSpringContextTests.java:161) ~[spring-test-5.3.12.jar:5.3.12]
	at jdk.internal.reflect.GeneratedMethodAccessor160.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:527) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128) ~[testng-7.4.0.jar:7.4.0]
	at java.util.ArrayList.forEach(ArrayList.java:1541) [?:?]
	at org.testng.TestRunner.privateRun(TestRunner.java:794) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestRunner.run(TestRunner.java:596) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.run(SuiteRunner.java:276) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuites(TestNG.java:1063) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.run(TestNG.java:1031) [testng-7.4.0.jar:7.4.0]
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418) [surefire-booter-2.22.2.jar:2.22.2]
Caused by: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:220) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	... 48 more
Caused by: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:349) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	... 48 more
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:455) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:447) ~[?:?]
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227) ~[?:?]
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:80) ~[?:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:344) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	... 48 more
2022-08-30 12:40:41.075 WARN [main] org.springframework.test.context.TestContextManager - Caught exception while invoking &apos;beforeTestMethod&apos; callback on TestExecutionListener [org.springframework.test.context.web.ServletTestExecutionListener@8c6da7c] for test method [public void com.fdc.core.cloudsort.service.auth.CloudSortJwtStorageIntTest.init_ok()] and test instance [com.fdc.core.cloudsort.service.auth.CloudSortJwtStorageIntTest@7cf162bc]
java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:132) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.support.DefaultTestContext.getApplicationContext(DefaultTestContext.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:190) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.beforeTestMethod(ServletTestExecutionListener.java:145) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.TestContextManager.beforeTestMethod(TestContextManager.java:293) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.springTestContextBeforeTestMethod(AbstractTestNGSpringContextTests.java:161) ~[spring-test-5.3.12.jar:5.3.12]
	at jdk.internal.reflect.GeneratedMethodAccessor160.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:527) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128) ~[testng-7.4.0.jar:7.4.0]
	at java.util.ArrayList.forEach(ArrayList.java:1541) [?:?]
	at org.testng.TestRunner.privateRun(TestRunner.java:794) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestRunner.run(TestRunner.java:596) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.run(SuiteRunner.java:276) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuites(TestNG.java:1063) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.run(TestNG.java:1031) [testng-7.4.0.jar:7.4.0]
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418) [surefire-booter-2.22.2.jar:2.22.2]
Caused by: com.github.tomakehurst.wiremock.common.FatalStartupException: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:157) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 41 more
Caused by: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:220) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 41 more
Caused by: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:349) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 41 more
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:455) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:447) ~[?:?]
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227) ~[?:?]
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:80) ~[?:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:344) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 41 more
</stdout>
          <failedSince>318</failedSince>
        </case>
        <case>
          <duration>0.0</duration>
          <className>com.fdc.core.cloudsort.service.auth.CloudSortJwtStorageIntTest</className>
          <testName>get_expiredToken_ok</testName>
          <skipped>true</skipped>
          <skippedMessage>Failed to load ApplicationContext</skippedMessage>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.0</duration>
          <className>com.fdc.core.cloudsort.service.auth.CloudSortJwtStorageIntTest</className>
          <testName>get_ok</testName>
          <skipped>true</skipped>
          <skippedMessage>Failed to load ApplicationContext</skippedMessage>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.0</duration>
          <className>com.fdc.core.cloudsort.service.auth.CloudSortJwtStorageIntTest</className>
          <testName>init_ok</testName>
          <skipped>true</skipped>
          <skippedMessage>Failed to load ApplicationContext</skippedMessage>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.227</duration>
          <className>com.fdc.core.cloudsort.service.CloudSortRestClientIntTest</className>
          <testName>springTestContextPrepareTestInstance</testName>
          <skipped>false</skipped>
          <errorStackTrace>java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: com.github.tomakehurst.wiremock.common.FatalStartupException: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
Caused by: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
Caused by: java.io.IOException: Failed to bind to /0.0.0.0:8191
Caused by: java.net.BindException: Address already in use
</errorStackTrace>
          <errorDetails>Failed to load ApplicationContext</errorDetails>
          <stdout>
  .   ____          _            __ _ _
 /\\ / ___&apos;_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | &apos;_ | &apos;_| | &apos;_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  &apos;  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::                (v2.5.6)

2022-08-30 12:40:41.231 Verbose logging enabled
2022-08-30 12:40:41.243 WARN [main] wiremock.org.eclipse.jetty.server.AbstractConnector - Acceptors should be &lt;= availableProcessors: NetworkTrafficServerConnector@42f3dac6{HTTP/1.1, (http/1.1, h2c)}{0.0.0.0:0}
2022-08-30 12:40:41.283 ERROR [main] org.springframework.boot.SpringApplication - Application run failed
com.github.tomakehurst.wiremock.common.FatalStartupException: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:157) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.support.DefaultTestContext.getApplicationContext(DefaultTestContext.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:190) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.prepareTestInstance(ServletTestExecutionListener.java:132) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:248) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.springTestContextPrepareTestInstance(AbstractTestNGSpringContextTests.java:149) ~[spring-test-5.3.12.jar:5.3.12]
	at jdk.internal.reflect.GeneratedMethodAccessor300.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122) ~[testng-7.4.0.jar:7.4.0]
	at java.util.ArrayList.forEach(ArrayList.java:1541) [?:?]
	at org.testng.TestRunner.privateRun(TestRunner.java:794) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestRunner.run(TestRunner.java:596) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.run(SuiteRunner.java:276) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuites(TestNG.java:1063) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.run(TestNG.java:1031) [testng-7.4.0.jar:7.4.0]
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418) [surefire-booter-2.22.2.jar:2.22.2]
Caused by: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:220) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	... 42 more
Caused by: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:349) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	... 42 more
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:455) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:447) ~[?:?]
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227) ~[?:?]
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:80) ~[?:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:344) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	... 42 more
2022-08-30 12:40:41.291 ERROR [main] org.springframework.test.context.TestContextManager - Caught exception while allowing TestExecutionListener [org.springframework.test.context.web.ServletTestExecutionListener@46c03fe4] to prepare test instance [com.fdc.core.cloudsort.service.CloudSortRestClientIntTest@766a4535]
java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:132) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.support.DefaultTestContext.getApplicationContext(DefaultTestContext.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:190) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.prepareTestInstance(ServletTestExecutionListener.java:132) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:248) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.springTestContextPrepareTestInstance(AbstractTestNGSpringContextTests.java:149) ~[spring-test-5.3.12.jar:5.3.12]
	at jdk.internal.reflect.GeneratedMethodAccessor300.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122) ~[testng-7.4.0.jar:7.4.0]
	at java.util.ArrayList.forEach(ArrayList.java:1541) [?:?]
	at org.testng.TestRunner.privateRun(TestRunner.java:794) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestRunner.run(TestRunner.java:596) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.run(SuiteRunner.java:276) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuites(TestNG.java:1063) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.run(TestNG.java:1031) [testng-7.4.0.jar:7.4.0]
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418) [surefire-booter-2.22.2.jar:2.22.2]
Caused by: com.github.tomakehurst.wiremock.common.FatalStartupException: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:157) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 35 more
Caused by: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:220) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 35 more
Caused by: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:349) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 35 more
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:455) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:447) ~[?:?]
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227) ~[?:?]
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:80) ~[?:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:344) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 35 more
</stdout>
          <failedSince>318</failedSince>
        </case>
        <case>
          <duration>0.455</duration>
          <className>com.fdc.core.cloudsort.service.CloudSortRestClientIntTest</className>
          <testName>springTestContextBeforeTestMethod</testName>
          <skipped>false</skipped>
          <errorStackTrace>java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: com.github.tomakehurst.wiremock.common.FatalStartupException: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
Caused by: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
Caused by: java.io.IOException: Failed to bind to /0.0.0.0:8191
Caused by: java.net.BindException: Address already in use
</errorStackTrace>
          <errorDetails>Failed to load ApplicationContext</errorDetails>
          <stdout>
  .   ____          _            __ _ _
 /\\ / ___&apos;_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | &apos;_ | &apos;_| | &apos;_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  &apos;  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::                (v2.5.6)

2022-08-30 12:40:41.404 Verbose logging enabled
2022-08-30 12:40:41.412 WARN [main] wiremock.org.eclipse.jetty.server.AbstractConnector - Acceptors should be &lt;= availableProcessors: NetworkTrafficServerConnector@76226519{HTTP/1.1, (http/1.1, h2c)}{0.0.0.0:0}
2022-08-30 12:40:41.472 ERROR [main] org.springframework.boot.SpringApplication - Application run failed
com.github.tomakehurst.wiremock.common.FatalStartupException: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:157) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.support.DefaultTestContext.getApplicationContext(DefaultTestContext.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:190) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.beforeTestMethod(ServletTestExecutionListener.java:145) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.TestContextManager.beforeTestMethod(TestContextManager.java:293) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.springTestContextBeforeTestMethod(AbstractTestNGSpringContextTests.java:161) ~[spring-test-5.3.12.jar:5.3.12]
	at jdk.internal.reflect.GeneratedMethodAccessor160.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:527) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128) ~[testng-7.4.0.jar:7.4.0]
	at java.util.ArrayList.forEach(ArrayList.java:1541) [?:?]
	at org.testng.TestRunner.privateRun(TestRunner.java:794) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestRunner.run(TestRunner.java:596) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.run(SuiteRunner.java:276) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuites(TestNG.java:1063) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.run(TestNG.java:1031) [testng-7.4.0.jar:7.4.0]
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418) [surefire-booter-2.22.2.jar:2.22.2]
Caused by: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:220) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	... 48 more
Caused by: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:349) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	... 48 more
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:455) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:447) ~[?:?]
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227) ~[?:?]
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:80) ~[?:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:344) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	... 48 more
2022-08-30 12:40:41.474 WARN [main] org.springframework.test.context.TestContextManager - Caught exception while invoking &apos;beforeTestMethod&apos; callback on TestExecutionListener [org.springframework.test.context.web.ServletTestExecutionListener@46c03fe4] for test method [public void com.fdc.core.cloudsort.service.CloudSortRestClientIntTest.auth_ok()] and test instance [com.fdc.core.cloudsort.service.CloudSortRestClientIntTest@766a4535]
java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:132) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.support.DefaultTestContext.getApplicationContext(DefaultTestContext.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:190) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.beforeTestMethod(ServletTestExecutionListener.java:145) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.TestContextManager.beforeTestMethod(TestContextManager.java:293) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.springTestContextBeforeTestMethod(AbstractTestNGSpringContextTests.java:161) ~[spring-test-5.3.12.jar:5.3.12]
	at jdk.internal.reflect.GeneratedMethodAccessor160.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:527) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128) ~[testng-7.4.0.jar:7.4.0]
	at java.util.ArrayList.forEach(ArrayList.java:1541) [?:?]
	at org.testng.TestRunner.privateRun(TestRunner.java:794) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestRunner.run(TestRunner.java:596) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.run(SuiteRunner.java:276) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuites(TestNG.java:1063) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.run(TestNG.java:1031) [testng-7.4.0.jar:7.4.0]
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418) [surefire-booter-2.22.2.jar:2.22.2]
Caused by: com.github.tomakehurst.wiremock.common.FatalStartupException: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:157) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 41 more
Caused by: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:220) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 41 more
Caused by: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:349) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 41 more
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:455) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:447) ~[?:?]
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227) ~[?:?]
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:80) ~[?:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:344) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 41 more
</stdout>
          <failedSince>318</failedSince>
        </case>
        <case>
          <duration>0.278</duration>
          <className>com.fdc.core.cloudsort.service.CloudSortRestClientIntTest</className>
          <testName>springTestContextBeforeTestMethod</testName>
          <skipped>false</skipped>
          <errorStackTrace>java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: com.github.tomakehurst.wiremock.common.FatalStartupException: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
Caused by: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
Caused by: java.io.IOException: Failed to bind to /0.0.0.0:8191
Caused by: java.net.BindException: Address already in use
</errorStackTrace>
          <errorDetails>Failed to load ApplicationContext</errorDetails>
          <stdout>
  .   ____          _            __ _ _
 /\\ / ___&apos;_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | &apos;_ | &apos;_| | &apos;_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  &apos;  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::                (v2.5.6)

2022-08-30 12:40:41.681 Verbose logging enabled
2022-08-30 12:40:41.697 WARN [main] wiremock.org.eclipse.jetty.server.AbstractConnector - Acceptors should be &lt;= availableProcessors: NetworkTrafficServerConnector@6b462f42{HTTP/1.1, (http/1.1, h2c)}{0.0.0.0:0}
2022-08-30 12:40:41.753 ERROR [main] org.springframework.boot.SpringApplication - Application run failed
com.github.tomakehurst.wiremock.common.FatalStartupException: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:157) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.support.DefaultTestContext.getApplicationContext(DefaultTestContext.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:190) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.beforeTestMethod(ServletTestExecutionListener.java:145) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.TestContextManager.beforeTestMethod(TestContextManager.java:293) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.springTestContextBeforeTestMethod(AbstractTestNGSpringContextTests.java:161) ~[spring-test-5.3.12.jar:5.3.12]
	at jdk.internal.reflect.GeneratedMethodAccessor160.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:527) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128) ~[testng-7.4.0.jar:7.4.0]
	at java.util.ArrayList.forEach(ArrayList.java:1541) [?:?]
	at org.testng.TestRunner.privateRun(TestRunner.java:794) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestRunner.run(TestRunner.java:596) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.run(SuiteRunner.java:276) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuites(TestNG.java:1063) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.run(TestNG.java:1031) [testng-7.4.0.jar:7.4.0]
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418) [surefire-booter-2.22.2.jar:2.22.2]
Caused by: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:220) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	... 48 more
Caused by: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:349) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	... 48 more
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:455) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:447) ~[?:?]
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227) ~[?:?]
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:80) ~[?:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:344) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	... 48 more
2022-08-30 12:40:41.767 WARN [main] org.springframework.test.context.TestContextManager - Caught exception while invoking &apos;beforeTestMethod&apos; callback on TestExecutionListener [org.springframework.test.context.web.ServletTestExecutionListener@46c03fe4] for test method [public void com.fdc.core.cloudsort.service.CloudSortRestClientIntTest.manifest_ok()] and test instance [com.fdc.core.cloudsort.service.CloudSortRestClientIntTest@766a4535]
java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:132) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.support.DefaultTestContext.getApplicationContext(DefaultTestContext.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:190) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.web.ServletTestExecutionListener.beforeTestMethod(ServletTestExecutionListener.java:145) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.TestContextManager.beforeTestMethod(TestContextManager.java:293) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.springTestContextBeforeTestMethod(AbstractTestNGSpringContextTests.java:161) ~[spring-test-5.3.12.jar:5.3.12]
	at jdk.internal.reflect.GeneratedMethodAccessor160.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:527) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146) ~[testng-7.4.0.jar:7.4.0]
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128) ~[testng-7.4.0.jar:7.4.0]
	at java.util.ArrayList.forEach(ArrayList.java:1541) [?:?]
	at org.testng.TestRunner.privateRun(TestRunner.java:794) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestRunner.run(TestRunner.java:596) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunner.run(SuiteRunner.java:276) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53) [testng-7.4.0.jar:7.4.0]
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.runSuites(TestNG.java:1063) [testng-7.4.0.jar:7.4.0]
	at org.testng.TestNG.run(TestNG.java:1031) [testng-7.4.0.jar:7.4.0]
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146) [surefire-testng-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126) [surefire-booter-2.22.2.jar:2.22.2]
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418) [surefire-booter-2.22.2.jar:2.22.2]
Caused by: com.github.tomakehurst.wiremock.common.FatalStartupException: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:157) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 41 more
Caused by: java.lang.RuntimeException: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:220) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 41 more
Caused by: java.io.IOException: Failed to bind to /0.0.0.0:8191
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:349) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 41 more
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:455) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:447) ~[?:?]
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227) ~[?:?]
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:80) ~[?:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:344) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.server.Server.doStart(Server.java:401) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at wiremock.org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.jetty9.JettyHttpServer.start(JettyHttpServer.java:218) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.github.tomakehurst.wiremock.WireMockServer.start(WireMockServer.java:155) ~[wiremock-jre8-standalone-2.33.2.jar:?]
	at com.fdc.core.configuration.WireMockInitializer.initialize(WireMockInitializer.java:31) ~[test-classes/:?]
	at org.springframework.boot.SpringApplication.applyInitializers(SpringApplication.java:634) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.prepareContext(SpringApplication.java:403) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:337) ~[spring-boot-2.5.6.jar:2.5.6]
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:123) ~[spring-boot-test-2.5.6.jar:2.5.6]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99) ~[spring-test-5.3.12.jar:5.3.12]
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124) ~[spring-test-5.3.12.jar:5.3.12]
	... 41 more
</stdout>
          <failedSince>318</failedSince>
        </case>
        <case>
          <duration>0.0</duration>
          <className>com.fdc.core.cloudsort.service.CloudSortRestClientIntTest</className>
          <testName>auth_ok</testName>
          <skipped>true</skipped>
          <skippedMessage>Failed to load ApplicationContext</skippedMessage>
          <failedSince>0</failedSince>
        </case>
        <case>
          <duration>0.0</duration>
          <className>com.fdc.core.cloudsort.service.CloudSortRestClientIntTest</className>
          <testName>manifest_ok</testName>
          <skipped>true</skipped>
          <skippedMessage>Failed to load ApplicationContext</skippedMessage>
          <failedSince>0</failedSince>
        </case>
      </cases>
    </suite>
  </suites>
  <duration>199.918</duration>
  <keepLongStdio>false</keepLongStdio>
</result>